modules/swagger-codegen/src/main/resources/typescript-angular/api.module.mustache

```diff
@@ -1,6 +1,6 @@
 import { NgModule, ModuleWithProviders } from '@angular/core';
 import { CommonModule } from '@angular/common';
-import { HttpModule } from '@angular/http';
+import { HttpClientModule } from '@angular/common/http';
```

Comment:
it should be distinguished between http client module and http module. See e.g. https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/typescript-angular/encoder.mustache#L1
moreover, the samples should be re-generated